src/pages/aura/pricing.md

```
@@ -18,27 +18,27 @@ Start exploring blockchain development at no cost. Perfect for hobbyists, early-
 
 **Ideal For**: Startups, scaling projects, or developers who want cost efficiency without committing to an upfront plan.
 
-| Feature            | RPS | Price/Req MPLX | Price/Req USD |
-| ------------------ | --- | -------------- | ------------- |
-| Solana DAS API     | 75  | 0.000107       | 0.0000380     |
-| Eclipse DAS API    | 75  | 0.000107       | 0.0000380     |
-| Solana RPC         | 50  | 0.000013       | 0.0000048     |
-| Eclipse RPC        | 50  | 0.000013       | 0.0000048     |
-| getProgramAccounts | 25  | 0.000134       | 0.0000475     |
+| Feature            | RPS | Price/Req MPLX |
+| ------------------ | --- | -------------- | 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)